components/Header/Header.module.scss

```diff
@@ -130,13 +135,17 @@
         display: none;
     }
 }
-
-@media (max-width: 992px) {
-    .logo {
+.logo{
+    background-image: url(/img/deriv.svg);
+    width: 180px;
+    height: 73px;
+    @media (max-width: 992px) {
```

Comment:
replace media query with [data-state~=state.responsive....etc] just like in https://github.com/deriv-pocs/deriv-developers-portal/pull/59/files#diff-0c4f92e7b1d221127bc3e6e9839b06fbc203588b8e721d640e45b8c72db98973R126-R129